### PR TITLE
[rustpotterks] Fix for Windows, avoid load library multiple times

### PR DESCRIPTION
**Pull request by GiviMAD**
Hello, 
I would like to merge these minor fix as a user from the forum has let me know that the library fails on windows after the first run.

This commit is not need in the main branch because the new library version already fixes this.

Best regards!

### CODE REVIEW

**Comment by GiviMAD** [comment]
Hello, I'm having some problems with spotlight in files I've not touch, any idea? thank you for the help.

File: bundles/org.openhab.voice.rustpotterks/src/main/java/org/openhab/voice/rustpotterks/internal/RustpotterKSService.java [hunk 1]
```diff
@@ -61,6 +61,7 @@
 @ConfigurableService(category = SERVICE_CATEGORY, label = SERVICE_NAME
         + " Keyword Spotter", description_uri = SERVICE_CATEGORY + ":" + SERVICE_ID)
 public class RustpotterKSService implements KSService {
+    private static boolean libraryLoaded;
```
Comment (lolodomo) [inline bundles/org.openhab.voice.rustpotterks/src/main/java/org/openhab/voice/rustpotterks/internal/RustpotterKSService.java:64]: Please remove "static".

**Review by lolodomo — CHANGES_REQUESTED** [review]

**Review by lolodomo — APPROVED** [review]
LGTM, thank you

**Comment by lolodomo** [comment]
Please rebase your branch, it looks like you have an "invalid" pom.xml file in your branch (badly formatted).

**Comment by GiviMAD** [comment]
When I run rebase it told me my branch is is up to date with 4.0.x.

If I run spotless:apply in the root of my folder it changes all the pom.xml files in the project on the first line.
It added a break line at the begging of each project tag.
```xml
<?xml version="1.0" encoding="UTF-8" standalone="no"?><project ...
```
Is it because I created my branch from the 4.0.x branch? I did it like that because the code was already different on main and the issue doesn't exist there.

**Comment by lolodomo** [comment]
> Is it because I created my branch from the 4.0.x branch? I did it like that because the code was already different on main and the issue doesn't exist there.

I did not take about that ! Normally we should avoid merging in 4.0.x directly.
Is your fix really critical ?

@jlaur: an advice how to proceed ?

**Comment by GiviMAD** [comment]
> Is your fix really critical ?

No, it prevents using it on windows and no one has complain until now, and it's already fixed on 4.1.0 because of the library update. And the person how wrote me about it is going to test it in the milestone instead, so I can just close it.

**Review by jlaur — APPROVED** [review]
Thanks!

**Comment by jlaur** [comment]
> @jlaur: an advice how to proceed ?

We can accept PR's targeting a patch branch directly when cherry-picking is impossible - for example due to conflicts, or like in this case irrelevancy because of other factors.

**Comment by GiviMAD** [comment]
Thank you for the review!

**Pull request merged**